### DISCUSSION TITLE
Installing gleam on windows

### DISCUSSION THREAD

**Discussion by shishini** [gleam-lang/gleam#3008] [General]
Hi 

I installed erlang and gleam using the following commands
```
winget install erlang
winget install gleam
```
winget is not mentioned on the [gleam.run](https://gleam.run/getting-started/installing/) but it works

I then followed the instruction for rebar3 , but obviously I get this error 

```
PS C:\> rebar3 --help
'escript.exe' is not recognized as an internal or external command,
operable program or batch file.
```
 
 because [Beginning with R9C, the Windows installer does not add Erlang to the system wide path. If one wants to have Erlang in the path, one has to add it by hand.](https://erlang.org/documentation/doc-5.10.1/doc/installation_guide/INSTALL-WIN32.html#id76867)
 
 My questions are
1.  Why did the erlang installer stop adding erlang to the system path or even user path, is there a good reason for this, should I avoid it
2. rebar3 need erlang bin directory on the path, so why isnt this mentioned anywhere
3. the erlang bin folder have some .BOOT files, and I dont know if i should have those on my path 

Any advice, should i just add the erlang bin folder to the path, or is there a better option

**Comment by lpil** [comment]
Hello! I'm afraid we are unable to offer any help with third party packages. We support the officially documented entrypoint for rebar3 and if a package you are using does something that differs from what the rebar3 documentation says to do to install it then currently we are unable to support that.
